### PR TITLE
[v1] Support radians in interpolateNode()

### PR DESCRIPTION
**Pull request by javascripter**
## Description

This PR will add support for strings with radians in interpolate's outputRange.
Currently code like this works on iOS but crashes on Android.

ref.
https://github.com/software-mansion/react-native-reanimated/issues/1436#issuecomment-726779954

```typescript
Animated.interpolate(animatedValue, {
  inputRange: [0, 1],
  outputRange: ['0rad', `${Math.PI}rad`]
})
```

## Changes
- Added code to convert strings with radians to number
- Added a test in `interpolateNodeTest()` function

## Test code and steps to reproduce

Added test case

## Checklist

- [x] Included code example that can be used to test this change
- [ ] Updated TS types
- [ ] Added TS types tests
- [x] Added unit / integration tests
- [ ] Updated documentation
- [x] Ensured that CI passes

### CODE REVIEW

File: react-native-reanimated-tests.tsx [hunk 1]
```diff
@@ -303,16 +303,16 @@ function AnimatedGestureHandlerTest() {
...
+  const gestureHandler = useAnimatedGestureHandler<PinchGestureHandlerGestureEvent>(
+    {
+      onActive: (event) => {
+        x.value = event.scale;
+      },
+      onEnd: () => {
+        x.value = withTiming(1);
+      },
+    }
+  );
```
Comment (jakub-gonet) [inline react-native-reanimated-tests.tsx:315]: This change is unrelated.

**Review by jakub-gonet — APPROVED** [review]
If you can, please edit examples from Reanimated 1 to allow testing this change, e.g. [this one](https://github.com/software-mansion/react-native-reanimated/blob/master/Example/reanimated1/src/interpolate/Basic.js).

Other than that, looks great, thanks.

**Comment by javascripter** [comment]
Thanks for reviewing! Removed unrelated style changes and updated Example to allow testing this. @jakub-gonet

https://user-images.githubusercontent.com/13040/111024020-b7f3e880-841f-11eb-94c6-ff612f56ac16.mov

**Review by jakub-gonet — APPROVED** [review]
